<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-6RR0I9J" failures="1" tests="9" name="com.qc.test.LoginTest2" time="21.958" errors="0" timestamp="2022-04-05T12:51:25 IST" skipped="0">
  <testcase classname="com.qc.test.LoginTest2" name="doLogin" time="2.750"/>
  <system-out/>
  <testcase classname="com.qc.test.LoginTest2" name="doLogin" time="2.446"/>
  <system-out/>
  <testcase classname="com.qc.test.LoginTest2" name="doLogin" time="2.504"/>
  <system-out/>
  <testcase classname="com.qc.test.LoginTest2" name="CloseBrowser" time="2.451">
    <failure message="The following asserts failed:
	expected [Queue codes | Log in] but found [Queue Codes | Log in],
	expected [Queue codes | Log in] but found [Queue Codes | Log in],
	expected [Queue codes | Log in] but found [Queue Codes | Log in],
	expected [Queue codes | Log in] but found [Queue Codes | Log in],
	expected [Queue codes | Log in] but found [Queue Codes | Log in],
	expected [Queue codes | Log in] but found [Queue Codes | Log in],
	expected [Queue Codes | Dashboar] but found [Queue Codes | Dashboard],
	expected [Queue codes | Log in] but found [Queue Codes | Log in]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Queue codes | Log in] but found [Queue Codes | Log in],
expected [Queue codes | Log in] but found [Queue Codes | Log in],
expected [Queue codes | Log in] but found [Queue Codes | Log in],
expected [Queue codes | Log in] but found [Queue Codes | Log in],
expected [Queue codes | Log in] but found [Queue Codes | Log in],
expected [Queue codes | Log in] but found [Queue Codes | Log in],
expected [Queue Codes | Dashboar] but found [Queue Codes | Dashboard],
expected [Queue codes | Log in] but found [Queue Codes | Log in]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.qc.test.BaseClass2.tearDown(BaseClass2.java:41)
at com.qc.test.LoginTest2.CloseBrowser(LoginTest2.java:52)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:344)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- CloseBrowser -->
  <system-out/>
  <testcase classname="com.qc.test.LoginTest2" name="doLogin" time="2.892"/>
  <system-out/>
  <testcase classname="com.qc.test.LoginTest2" name="doLogin" time="2.445"/>
  <system-out/>
  <testcase classname="com.qc.test.LoginTest2" name="doLogin" time="2.469"/>
  <system-out/>
  <testcase classname="com.qc.test.LoginTest2" name="doLogin" time="2.608"/>
  <system-out/>
  <testcase classname="com.qc.test.LoginTest2" name="doLogin" time="1.393"/>
  <system-out/>
</testsuite> <!-- com.qc.test.LoginTest2 -->
